packages/components/src/Components/FilterHooks/tagFilterHook.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from 'react';
+import { constructGroups, mapGroups } from './constants';
+import { Spinner } from '@patternfly/react-core/dist/esm/components/Spinner';
```

Comment:
We should not directly reference `esm` builds. https://github.com/RedHatInsights/frontend-components/pull/700 deals with proper build imports.